src/plotting/README.md

```diff
@@ -0,0 +1,57 @@
+Scripts for plotting data contained in either the sqlite or mysql databases.
+
+plot_barcode_family.py and plot_line_family.py are built to use sqlite as the backend. 
+
+Scripts which contain "mysql" in title reference the mysql database.
+
+
+# Usage
+
+Each script is designed to be called independently and produce a single plot. Calls are made to be to python with flags as necessary as defined below.
+
+Example call:
+```sh
+python3 plot_mysql_dir_sensor_sat.py --sidb ../satinfo 
+```
+
+## Flag options 
+
+--sidb provides the location of the satinfo folder for plots which go down to the satellite level to include the black/white list 
```

Comment:
```suggestion
--sidb provides the location of the satellite info (satinfo) folder for plots which go down to the satellite level to include the black/white list
```

Reply:
Possibly out of scope, but could this a good place to document the meaning of "black/white list"? Or, is an external reference available that could be linked here? Just something to think about as the demand for documentation increases.

Reply:
this would not be the best place for that documentation, either in the repo where the satinfo files live (build gsinfo) or in the new webpage we are setting up about the archive would be more appropriate in my opinion